### DISCUSSION TITLE
How to move /home/linuxbrew to a different device

### DISCUSSION THREAD

**Discussion by ssteinerx** [Homebrew/discussions#3707] [Everyday usage] [answered]
The device on which the /home directory is mounted is full.  

Homebrew is taking up ~10GB that I could really use for "elbow room" while I clean things up.

How can I (temporarily or permanently) move /home/linuxbrew to a different device?

Can I just symlink it to somewhere else and move the files there?  That would seem easiest since I won't have to update startup files etc.

If I can't just symlink it, is there a documented procedure for hunting down all the places I'll need to change "/home/linuxbrew" to something else?

Thanks in advance for any help.

**Comment by gromgit** [comment, marked as answer]
I used symlinks myself when trying to port Homebrew to a Raspberry Pi, but I ran into various problems related to Homebrew wanted to "canonicalize" most symlinks when running (can't remember what they were off-hand now).

Since you're running Linux, a much better way to do this is to use [_bind mounts_](https://www.baeldung.com/linux/bind-mounts), in which you mount an arbitrary directory hierarchy on top of another arbitrary mount point. That's why my Raspberry Pi now has the following lines in `/etc/fstab`:
```
# external hard drive
LABEL=DATA1     /data1  auto    noatime 0       1
# bind-mount .../linuxbrew
/data1/linuxbrew        /home/linuxbrew none    bind
```
Now everything Homebrew-related is on the external disk, and no symlink confusion.

**Reply by ssteinerx** [reply]
Thank you!  Worked like a charm.  Should be part of the documentation...
